FishFate/www/views/about.html

```diff
@@ -16,105 +16,110 @@
       <h4>How do the fish make a random number?</h4>
       <ol class="rectangle-list">
          <li>
-           <a href="" ng-click="clickListItem(0)">Fish Setup <br>
+           <a href="" ng-click="clickListItem(0)"><b>Fish Setup</b>
              <div id="listItem0">
                <!-- add content here -->
-               more content
 
              </div>
            </a>
          </li>
-         <li><a href="" ng-click="clickListItem(1)">Fish Tracking</a></li>
+         <li>
+           <a href="" ng-click="clickListItem(1)"><b>Fish Tracking</b>
+             <div id="listItem1">
+               <img ng-src="../img/fishPosition.png" style="height: 50%; width: 50%; margin-left:22.5%;"><br><br>
+               Fish movements are tracked by their initial position on an x/y plane, where their
+               next position they move to will generate a binary number as
+               shown on the the picture above.
+             </div>
+           </a>
+         </li>
 
-         <li><a href="" ng-click="clickListItem(2)">Generating Randomness</a></li>
+         <li>
+           <a href="" ng-click="clickListItem(2)"><b>Generating Randomness</b>
+             <div id="listItem2">
+               <br>We read in a certain number of bits from the bitstream and then calculate the
+               distribution of the 1's and 0's.<br><br>From that we calculate the Shannon Entropy which tells
+               the amount of entropy in the bits. Entropy is then calculated
+               until there are 256 bits of entropy to be inputted into a SHA1 Hash Function.<br><br>The hash
+               function then outputs a 160 bit Hex number and finally this Hex number is converted to
+               a Binary number to send to the server.
+             </div>
+           </a>
+         </li>
 
-         <li><a href="" ng-click="clickListItem(3)">How Random?</a></li>
+         <li>
+           <a href="" ng-click="clickListItem(3)"><b>How Random?</b>
+             <div id="listItem3">
+               <img ng-src="{{images.about.graph}}" style="height:65%; width:65%; margin-left:17.5%;"class="stretch"><br>
```

Comment:
this css logic looks good, but you should move it over to the dedicated stylesheet for this view `about.css`